pkgs/development/libraries/qt-6/modules/qtvirtualkeyboard.nix

```diff
@@ -11,5 +11,4 @@ qtModule {
   qtInputs = [ qtbase qtdeclarative qtsvg ];
   propagatedBuildInputs = [ hunspell ];
   nativeBuildInputs = [ pkg-config ];
-  outputs = [ "out" ];
```

Comment:
I'm confused, how does this add a dev output?

Reply:
`qtModule` by default sets `output = [ "out" "dev" ];`